### PR TITLE
Feature/upgrade alpine

### PR DESCRIPTION
**Pull request by peteclark-ft**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,24 +1,18 @@
-FROM alpine:3.4
+FROM alpine:3.7
```
Comment (SCBbestof) [inline Dockerfile]: Aren't we using 3.8 ?

Comment (peteclark-ft) [reply Dockerfile]: We can't I'm afraid.. `alpine:3.8` bundles varnish `6.0.0-r01`, which is incompatible with the `vmod-basicauth` module we use. `alpine:3.7` bundles varnish `5.2.1-r01` which fortunately is compatible

Comment (peteclark-ft) [reply Dockerfile]: It's possible the `vmod-basicauth` might be upgraded at some point to support varnish 6, but not as yet

Comment (SCBbestof) [reply Dockerfile]: Oh. Ok.

But I think we should look into varnish 6.0.0 for the future.

**Pull request merged**